meshtastic/mt_config.py

```
@@ -13,27 +13,60 @@
+    )
+except (AttributeError, NameError, TypeError):
+    _module_annotations = globals().get("__annotations__", {})
+_annotated_state: frozenset = frozenset(
+    k for k in _module_annotations if k in _state_keys
+)
+if _module_annotations and _state_keys != _annotated_state:
+    raise AssertionError(
+        f"Drift between MODULE_STATE_DEFAULTS and type annotations — "
+        f"missing annotations: {_state_keys - _annotated_state}, "
+        f"missing defaults: {_annotated_state - _state_keys}"
+    )
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)